SonarQube.MSBuild.Tasks/Resources.resx

```diff
@@ -208,4 +208,7 @@ Error: {1}</value>
   <data name="BuildWrapper_WaitingForAttach" xml:space="preserve">
     <value>BuildWrapper: waiting for the build wrapper to attach to the current process ({0})...</value>
   </data>
+  <data name="BuildWrapper_MissingOutputDirectory" xml:space="preserve">
+    <value>BuildWrapper: internal error. Invalid analysis config file - the build wrapper output directory must be supplied</value>
```

Comment:
We shouldn't hit this unless something has gone wrong elsewhere in our code. There isn't much a user will be able to do; the message is just to help with diagnosis.